### ISSUE TITLE
Accessibility: Naming generic landmarks

### ISSUE THREAD

**Issue by banksashleym** [JacksonCollege/jc-theme-2015#208]
I have also enabled the [SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc?hl=en-US) accessibilty checker in chrome. One issue that has been coming up on all pages is: Naming Generic Landmarks

`<section id="site-content" style="z-index: 2147483646; position: relative;">...</section>`

**The required action is:**
The page contains an HTML5 <section> or a WAI-ARIA landmark of the type 'region'. These types of landmarks are very general and should be named to help users identify the purpose of the landmark.

Use either the WAI-ARIA attribue 'aria-label' or 'aria-labelledby' to name the landmark in a way that clearly explains the purpose of the landmark to all users.

Thanks!

**Comment by joelworsham** [comment]
Added some `aria-label`'s to the main site sections.
